src/Emogrifier/CssInliner.php

```diff
@@ -783,7 +783,9 @@ private function splitCssAndMediaQuery(string $css): array
      */
     private function copyInlinableCssToStyleAttribute(\DOMElement $node, array $cssRule)
     {
-        $newStyleDeclarations = $this->parseCssDeclarationsBlock((string)$cssRule['declarationsBlock']);
+        /** @var string $declarationsBlock */
```

Comment:
Does it need newlines in the DocBlock like
```php
/**
 * @var string $declarationsBlock
 */
```
or does it also work as a single line?

Reply:
Single-line doc blocks work just as fine.